### DISCUSSION TITLE
Is it possible to keep holding a key to execute the holding event?

### DISCUSSION THREAD

**Discussion by B1T3X** [kmonad/kmonad#512] [Q&A] [answered]
Hi,
I'm trying to map Caps+HJKL to the arrow keys.
I have created a layer called `arrows` where hjkl are the arrow keys and created the following alias:
```
(defalias 
  pressarrows (tap-hold-next-release 200 esc (around-next (layer-toggle arrows)))
)
```

I have mapped the `Caps` key to `@pressarrows` in the `arrows` layer and it works, when I tap `Caps` it results in `Escape`, when I hold it and press h/j/k/l it will press the corresponding arrow key. The thing is, to do this again, I have to let go of `Caps`, and then hold it again, which is kind of tiring if I want to press the same arrow key repeatedly.

Is there any way to make it so I won't have to release and re-hold `Caps`?

Thanks,
Orel

**Comment by slotThe** [comment, marked as answer]
I would guess you'd just need to remove that `around-next`.  The normal `layer-toggle` already has the functionality you are looking for.  

Also

> I have mapped the Caps key to @pressarrows in the arrows layer and it works,

You should map caps to `@pressarrows` in your base layer (not the `defsrc`), not the layer you want to go to!

**Reply by B1T3X** [reply]
Seems to have done the trick
Thank you so much!
